CONTRIBUTING.md

```diff
@@ -30,6 +30,10 @@ more documentation, look [here](https://guides.github.com/activities/forking/).
 
 Some pull request guidelines:
 
+- We use the [`black`](https://black.readthedocs.io/en/stable/index.html) auto-formatter
+  to enforce style conventions in Manticore. To ensure your code is properly 
+  formatted, run `black -t py36 -l 100 .` in the manticore directory before
```

Comment:
Make sure to update this when you've decided on the final line length in `.travis.yml`. Right now travis does `120` but here it says `100`

Reply:
You caught that much faster than I was expecting. My thought was to suggest 100, but only enforce 120. Thinking about it more, lingering 120-character lines in the repo would risk getting reformatted by later, unrelated pull requests. Perhaps I should rewrite the above command to only look at modified files, or we should simply pick one number and stick to it.

Reply:
I think one number would make things much simpler for both the CI check and contributing, but I can see how a few 120-character lines could read better in special circumstances.

Are there any flags that black will read to possibly allow certain lines to not be reformatted? I'm not too familiar with the options

Reply:
In other news, Travis has just reminded me that `black` will extend lines that have been unnecessarily shortened to fit the specified max line length, so we can only choose one number. I'll change it to 100 for now but I'm willing to hear arguments otherwise. Personally, I do find 100 to be a bit cramped, but the `black` docs raise the points that line lengths over ~100 characters start to cause problems for:
* Side-by-side code review
* Visually impaired people
* Presenting code on slides

We'll see how it goes

Reply:
I'd be up for 120 but 100 is also ok as I agree with the points. OTOH you don't use 120 lengthy lines that often, maybe for some documentation(?) or when you have too many scopes (but in this case u probably should move sth to a function/method if it's not a hot code)